### PR TITLE
chore(deps): bump shell and driver

### PR DESCRIPTION
**Pull request by mabaasit**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/connection-form/src/utils/csfle-handler.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export interface UpdateCsfleKmsAction {
 
 export interface UpdateCsfleKmsTlsAction {
   type: 'update-csfle-kms-tls-param';
-  kmsProvider: KMSProviderName;
+  kmsProvider: KMSTLSProviderName;
```
[the remark this answers is not in the document]
Comment (addaleax) [reply packages/connection-form/src/utils/csfle-handler.ts:43]: @mabaasit Yes. Do you want to open a PR against the driver for that?

**Review by addaleax — APPROVED** [review]
LGTM!

File: packages/connection-form/src/utils/csfle-handler.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (addaleax) [reply packages/connection-form/src/utils/csfle-handler.ts:43]: Actually, taking a second look – `KMSTLSProviderName` is currently just an alias for `string`, right?

[the remark this answers is not in the document]
Comment (mabaasit) [reply packages/connection-form/src/utils/csfle-handler.ts:43]: yes, it is. the difference probably lies in `KMSProviderName` being `PROVIDER_NAME:${string}` and `KMSTLSProviderName` being `PROVIDER_NAME | string`.

**Pull request merged**